packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts

```diff
@@ -0,0 +1,344 @@
+import { ISliderStyleProps, ISliderStyles, getFocusStyle, AnimationVariables, getGlobalClassNames } from '../..';
+import { HighContrastSelector } from '@uifabric/styling';
+
+const GlobalClassNames = {
+  root: 'ms-Slider',
+  container: 'ms-Slider-container',
+  showTransitions: 'ms-Slider-showTransitions',
+  slideBox: 'ms-Slider-slideBox',
+  line: 'ms-Slider-line',
+  thumb: 'ms-Slider-thumb',
+  active: 'ms-Slider-active',
+  inactive: 'ms-Slider-inactive',
+  value: 'ms-Slider-value'
+};
+
+export const getStyles = (props: ISliderStyleProps): ISliderStyles => {
+
+  const { className, theme, vertical, buttonClassName, showValue, disabled } = props;
+  const { palette } = theme;
+  const classNames = getGlobalClassNames(GlobalClassNames, theme);
+
+  // @TODO(keco): Do we need to apply .rootIsDisabled / .rootIsEnabled classes for backwards-compat?
+  return ({
+    root: [
+      classNames.root,
+      {
+        userSelect: 'none',
+        selectors: {
+          '&.titleLabel': {
+            padding: 0
+          }
+        }
+      },
+      vertical && [
+        {
+          marginRight: 8
+        }
+      ],
+      disabled && [
+        {
+          selectors: {
+            '.thumb': {
+              borderColor: palette.neutralTertiaryAlt,
+              selectors: {
+                [HighContrastSelector]: {
+                  borderColor: 'GrayText'
+                }
+              }
+            },
+            // @TODO(keco): Is this OK or should we still target .activeSection
+            '.ms-Slider-active': {
+              background: palette.neutralTertiaryAlt,
+              selectors: {
+                [HighContrastSelector]: {
+                  backgroundColor: 'GrayText',
+                  borderColor: 'GrayText'
+                }
+              }
+            },
+            // @TODO(keco): Is this OK or should we still target .inactiveSection
+            '.ms-Slider-inactive': {
+              background: palette.neutralLight,
+              selectors: {
+                [HighContrastSelector]: {
+                  backgroundColor: 'GrayText',
+                  borderColor: 'GrayText'
+                }
+              }
+            }
+          }
+        }
+      ],
+      className,
+      disabled ? 'ms-Slider-disabled' : 'ms-Slider-enabled', // @TODO(keco): encapsulate styles
```

Comment:
These don't seem to be used in `.scss` anywhere and are just helper selectors for extensibility.. is this an OK approach to apply them then?